pages/common/colordiff.md

```diff
@@ -0,0 +1,25 @@
+# colordiff
+
+> A tool to colorize diff diff output.
+> The Perl script colordiff is a wrapper for `diff` and produces the same output but with pretty 'syntax' highlighting. Colour schemes can be customized..
```

Comment:
Any reason syntax is in quotes? Also, let's remove "pretty" to remain objective. And the last sentence has 2 periods.

Reply:
No reason, I will remove it